fragment.go

```diff
@@ -312,7 +313,8 @@ func (f *Fragment) close() error {
 
 func (f *Fragment) closeStorage() error {
 	// Clear the storage bitmap so it doesn't access the closed mmap.
-	f.storage = roaring.NewBitmap()
+
+	//f.storage = roaring.NewBitmap()
```

Comment:
are we concerned about anything accessing the closed mmap now that we aren't doing this? Should be nil out all the container storage here to prevent this?

Reply:
nil - ing out the container storage would make it available for GC. i could add an extra pointer to the fragment to hold the reference during the invalid period that occurs after close and before open, but all that would accomplish is awareness of the unexpected act. In the current release, we are throwing away those actions that occur during that period.  I'm not even sure that can happen.